Makefile

```diff
@@ -0,0 +1,48 @@
+VERSION = v2.3.1.160220
```

Comment:
This can be determined from `addons/main/script_mod.hpp` using grep, sed and some makefile cleverness.